renderer/src/components/UpdateDescription/UpdateDescription.tsx

```diff
@@ -18,6 +18,8 @@ export default function UpdateDescription() {
   );
 
   const [isOpen, setIsOpen] = useState(update?.age === "new");
+  const slackDesktopLink = "slack://channel?team=T3PV37ANP&id=C069N5LUP3M";
```

Comment:
You can move these two constants to another file named constants.ts and import them from it. 
Example: SLACK_DESTOP_LINK = "slack://channel?team=T3PV37ANP&id=C069N5LUP3M";
And the same for another.